invenio_rdm_records/administration/views/oai.py

```diff
@@ -82,9 +96,23 @@ class OaiPmhCreateView(AdminResourceCreateView):
     list_view_name = "OAI-PMH"
 
     form_fields = {
-        "name": {"order": 1, "text": "Set name"},
-        "spec": {"order": 2, "text": "Set spec"},
-        "search_pattern": {"order": 3, "text": "Search query"},
+        "name": {
+            "order": 1,
```

Comment:
This is what leads me to think above should be `1`.

Values are fully repeated? is there a way to re-use?

Reply:
for now, no, however we want to create a way to do that in the future -  for this release it was nice to have